tensorflow/third_party/mkl/BUILD

```diff
@@ -0,0 +1,9 @@
+# This file is replaced on github with a real build rule.
```

Comment:
hmm, it looks like this is one level too low - it should be third_party/mkl/BUILD ?  same for third_party/mkl/build_defs.bzl